Cargo.toml

```diff
@@ -18,5 +18,5 @@ thiserror = "1.0"
 lazy_static = "1.4.0"
 log = "0.4"
 pretty_env_logger = "0.4"
-fang = { version = "0.9" , features = ["asynk"], default-features = false }
+fang = { git = "https://github.com/ayrat555/fang.git" , features = ["asynk"], default-features = false }
```

Comment:
you can also set `rev` (commit hash) to lock the version

Reply:
Does not matter because tomorrow or in a few days fang will be released and i will change it to that version fang release.